packages/core-logger/package.json

```diff
@@ -6,7 +6,8 @@
         "Brian Faust <brian@ark.io>"
     ],
     "license": "MIT",
-    "main": "dist/index.js",
+    "main": "dist/index",
+    "types": "dist/index",
```

Comment:
A proper type definition should be created instead of just using the `dist` files.

Reply:
Typescript already produces declarations for us, but barrel index.ts in the root folder of each module will work too.
So rather than "dist/index.d.ts"
I'll create an "./index.ts" that uses the barrel pattern to export the types. Will that work?

Reply:
Done. Can I safely assume that this will be the pattern to adopt across all modules?

Reply:
I'm getting strange issues using the approach. it doesn't like a ./index.d.ts exporting types, which cause some test failures since it can't resolve the types.

However, if I export the types via ./src/index.ts, then the tests run. /src/index.ts automatically generates a /dist/index.d.ts. Since this works, is there any reason why we can't use 'types: dist/index'?

Reply:
Nevermind, I've got it working. Will be committing shortly..